src/controllers/discussionController.ts

```diff
@@ -221,6 +222,29 @@ const joinDiscussion = async (
 
     discussion.participants.push(user.userId);
     await discussion.save();
+
+     const meetingTime = new Date(discussion.date).toLocaleString();
```

Comment:
you can create a new file called emailService.ts then move all the email logic into it, then you call the service here like:
emailService.sendEmail(user,discussion,meetingLink, emailType)

email: type: meeting/unsubscribed/...

Reply:
treated this? @JaneGH

Reply:
i have await sendEmail({
      toEmail: user.email,
      subject: `You've joined the discussion: ${discussion.title}`,
      textContent: "",
      htmlContent: htmlContent,
    }); where sendEmail function from  import { sendEmail } from "../services/mailjetService"